### PR TITLE
Add sensible values for auto note symbols. 

### PR DESCRIPTION
**Pull request by natsirt721**
#### Summary
 ```SUMMARY: Interface "Change autonote map symbols to unique values"```

#### Purpose of change
Related: #38648
Changes auto note map symbols from 'yellow x' to various unique color/character combinations relating to the map extra. Also adds a column to the auto note manager to indicate which symbol belongs to which extra.

First step to enable in-game user customization of auto note map symbols.

#### Testing
* Enable auto notes
* Modify the `*.ano.json` to show all the map extras in the auto note manager and reload the save
* Examine the symbols in the UI
* Walk around a bit and see unique symbols for map extras appear

### CODE REVIEW

File: data/json/overmap/map_extras.json [hunk 1]
```diff
@@ -1,3 +1,8 @@
+Cataclysm: Dark Days Ahead JSON Web Linting Tool
+This is a tool to help modders and editors of the open source game Cataclysm: Dark Days Ahead write JSON in the game's expected format.
+
+Paste some JSON into the field below and click "Lint" to run an autoformatter against it.
+
```
Comment (anothersimulacrum) [inline data/json/overmap/map_extras.json:5]: You took too much from the linter.

Comment (natsirt721) [reply data/json/overmap/map_extras.json:5]: RIP

File: data/json/overmap/map_extras.json [hunk 2]
```diff
@@ -406,3 +406,4 @@
     "generator": { "generator_method": "map_extra_function", "generator_id": "mx_grave" }
   }
 ]
+
```
Comment (anothersimulacrum) [inline data/json/overmap/map_extras.json:409]:
```suggestion
```

File: src/auto_note.cpp [hunk 3]
```diff
@@ -239,7 +240,7 @@ void auto_note_manager_gui::show()
 
         // Draw horizontal line and corner pieces of the table
         for( int x = 0; x < 78; x++ ) {
-            if( x == 60 ) {
+            if( x == 51 || x == 60) {
```
Comment (ZhilkinSerg) [inline src/auto_note.cpp:243]:
```suggestion
            if( x == 51 || x == 60 ) {
```

File: src/auto_note.cpp [hunk 4]
```diff
@@ -265,8 +267,8 @@ void auto_note_manager_gui::show()
         // Clear table
         for( int y = 0; y < iContentHeight; y++ ) {
             for( int x = 0; x < 79; x++ ) {
-                // The middle beam needs special treatment
-                if( x == 60 ) {
+                // The middle beams needs special treatment
+                if( x == 51 || x == 60) {
```
Comment (ZhilkinSerg) [inline src/auto_note.cpp:271]:
```suggestion
                if( x == 51 || x == 60 ) {
```

**Pull request merged**